src/renderer/object/instanced_model.rs

```diff
@@ -124,6 +148,8 @@ impl<M: ForwardMaterial> InstancedModel<M> {
             program.use_attribute_vec3("position", &self.mesh.position_buffer)?;
         }
         if program.requires_attribute("uv_coordinates") {
+            program.use_uniform_vec4("textureTransform", &self.texture_transform.to_vec4())?;
+            program.use_attribute_vec4_instanced("subt", &self.instance_buffer4)?;
```

Comment:
This is what was messing me up. I didn't even think to put the subt req inside the uv req.